internal/cmd/root.go

```diff
@@ -48,7 +48,10 @@ type rootCommand struct {
 
 // newRootCommand creates a root command with a default launcher
 func newRootCommand(gs *state.GlobalState) *rootCommand {
-	return newRootWithLauncher(gs, newLauncher(gs))
+	if gs.Env["OLD_RESOLUTION"] == "true" {
```

Comment:
```suggestion
	if gs.Env["K6_OLD_RESOLUTION"] == "true" {
```
I guess we want it. As this is not expected as part of the description, what is the purpose? Is it to have a fallaback method in case we have bugs in the current implementation?

Reply:
More or less - yes. Also, I didn't want to remove it while doing the development for easy testing. 

It is now the question - do we want it or do we think we should drop it, in which case this PR will have a bunch of more deleted lines.

At least currently this doesn't really cost us much. And in all cases I would want to drop it in the next version, if we do not have any issues. 

So the name likely wil be `K6_OLD_RESOLUTION_EXPERIMENTAL` or something similar to signal that it should not be depended on for long.